app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,29 @@
+class BookingsController < ApplicationController
+  def show
+  end
+
+  def new
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new(bookings_params)
+    @booking.user = current_user
+    @booking.space = Space.find(params[:space_id])
+    if @booking.save
+      redirect_to booking_path(@booking)
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @booking.destroy
```

Comment:
You need to find the booking, no?

Reply:
Let's redirect it back to bookings_path?